static-build/nemu/Dockerfile_cross

```diff
@@ -0,0 +1,65 @@
+FROM debian
```

Comment:
You should probably have a copyright header in here :-)

Reply:
@grahamwhaley ok, thanks.